src/common/enums/authorization.privilege.ts

```diff
@@ -13,7 +13,7 @@ export enum AuthorizationPrivilege {
   CREATE_ORGANIZATION = 'create-organization',
   READ_USERS = 'read-users',
   UPDATE_CANVAS = 'update-canvas',
-  UPDATE_LIFECYCLE = 'update-lifecycle',
+  UPDATE_INNOVATION_FLOW = 'update-lifecycle',
```

Comment:
Should we also rename the 'update-lifecycle' literal to 'update-innovation-flow'?